src/tasks/FishingTask.py

```
@@ -0,0 +1,390 @@
+            self.screenshot(f"fishing_control_failed_{round_index}")
+            return False
+
+        self.log_error(f"第 {round_index} 轮重试超时，结束本轮")
+        self.screenshot(f"fishing_round_timeout_{round_index}")
+        return False
+
+    def enter_fishing_scene(self) -> bool:
+        # 暂时不使用该功能
+        return True
+        if not self.is_fishing_entry():
+            self.log_error("未检测到钓鱼入口，请先站在钓点旁")
```

Ruff (0.15.11)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)